install/kubernetes/helm/istio/charts/sidecar-injector/templates/configmap.yaml

```diff
@@ -39,8 +39,7 @@ data:
         {{ "[[ if (isset .ObjectMeta.Annotations \"traffic.sidecar.istio.io/includeInboundPorts\") -]]" }}
         {{ "- \"[[ index .ObjectMeta.Annotations \"traffic.sidecar.istio.io/includeInboundPorts\"  ]]\"" }}
         {{ "[[ else -]]" }}
-        - "{{ .Values.includeInboundPorts }}"
-        {{ "[[ end -]]" }}
+        - {{ "[[ range .Spec.Containers -]][[ range .Ports -]][[ .ContainerPort -]], [[ end -]][[ end -]][[ end]]" }}
```

Comment:
should it be some if - what if they did specify includeInboundPorts ?

cc @ayj

Reply:
the if is above. If inboundPort is specified, it will use that value.

Reply:
Please check the generated file, it behaves as expected (see description of the PR).